src/components/App/App.js

```diff
@@ -1,24 +1,25 @@
-import React from 'react';
-import {
-    HelloWorld1,
-    HelloWorld2,
-    HelloWorld4,
-    HelloWorld3
-} from '../Helloworld/Helloworld';
-import Header from '../Header/Header';
-import InfoComponent from '../Info/InfoComponent';
-import './App.css';
+import React, { useState } from 'react';
+import ContentHolderComponent from '../ContentHolder/ContentHolderComponent';
+import { TopComponent } from '../TopComponent/TopComponent';
+import movieList from '../../Movies.json';
 
 export default function App() {
+    const [movies, setMovies] = useState(movieList);
+
+    const onSearch = (query) => {
+        const filteredMovies = query.trim()
```

Comment:
If you use expression more than 1 time pls put it into a separate variable
I speak about `query.trim()`